manifests/charts/global.yaml

```diff
@@ -264,6 +259,14 @@ global:
       maxNumberOfAnnotations: 200
       # The global default max number of message events per span.
       maxNumberOfMessageEvents: 200
+    tlsSettings:
+      mode: DISABLE # DISABLE, SIMPLE, MUTUAL, ISTIO_MUTUAL
+      clientCertificate: # example: /etc/istio/tracer/cert-chain.pem
+      privateKey:        # example: /etc/istio/tracer/key.pem
+      caCertificates:    # example: /etc/istio/tracer/root-cert.pem
+      sni:               # example: tracer.somedomain
+      subjectAltNames: []
+      # - tracer.somedomain
```

Comment:
We shouldn't be adding these here afaik, we should move on to using Operator APIs @howardjohn correct me if I'm wrong.

Reply:
Yes you can directly set mesh config now. So this would be 
```
meshConfig:
  tracer:
    tlsSettings: ...
```